exercises/tasks/task_06.py

```diff
@@ -1,10 +1,10 @@
 from exercises.tasks.resources.resource_06_12 import KiwiPage, SearchResultPage
 
 
-# Searching with additional transportation options shows results cheaper than 1 000 CZK
+# Searching with additional transportation options shows results cheaper than 2000 CZK
```

Comment:
`2000` -> `2 000` (pls, add a space ` ` as thousands separator to keep it consistent with `exercises/solutions/solution_06.py`; same below on row 31).